cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -213,6 +213,8 @@
     protected Map<String, String> templateExternalBootstrapIngressAnnotations;
     protected Map<String, String> templatePerPodIngressLabels;
     protected Map<String, String> templatePerPodIngressAnnotations;
+    protected Map<String, String> templateClusterCALabels;
```

Comment:
Maybe this would be better?

```suggestion
    protected Map<String, String> templateClusterCACertLabels;
```